docker-compose.yml

```diff
@@ -262,6 +262,8 @@ services:
       - GRAFANA_SERVICE_HOST
       - JAEGER_SERVICE_PORT
       - JAEGER_SERVICE_HOST
+      - OTEL_COLLECTOR_HOST
```

Comment:
I think these environment variables belong in the `# Collector` section instead.  Though only used by Envoy, we should keep these vars with the other similar ones.

Reply:
@puckpuck I think you meant in the `.env` file, right?

Reply:
yes I meant in the .env file